### DISCUSSION TITLE
Dramatic changes to chunks after upgrading from Gatsby 4 to 5

### DISCUSSION THREAD

**Discussion by Undistraction** [gatsbyjs/gatsby#38375] [Help]
I've upgraded a site from `4.x` to `5.11.0` and have seen the following changes to bundles:
- Total bundle size has increased 2.5 times.
- Tree-shaking no longer seems to be working. 

I've been through all of the [release notes](https://www.gatsbyjs.com/docs/reference/release-notes/) for the `5.x` versions and don't see any mentions of changes that will affect chunking (other than the Slice API).

I'm not sure where to start trying to debug this as I have made the bare minimum of changes required to upgrade and have not changed anything that should lead to such a dramatic change. 

Any help would really be appreciated.

**Comment by Undistraction** [comment]
Silly mistake. Nothing to learn from this.

**Reply by tiimgreen** [reply]
@Undistraction what was the problem? Also seeing something similar
